app/services/admin_notices/dismiss.rb

```diff
@@ -17,7 +22,7 @@ def invalid_access(guardian:)
   end
 
   def fetch_admin_notice(params:)
-    AdminNotice.find_by(id: params[:id])
+    AdminNotice.find_by(id: params.id)
```

Comment:
I added contracts where they were missing, that way `params` are accessed the same way everywhere. I guess we should probably add as a guideline that if you have params, they should always be validated.